src/Command/GenerateThumbnail.php

```diff
@@ -0,0 +1,110 @@
+<?php
```

Comment:
Je n'ai pas bien compris le cas d'utilisation de créer une miniature. Lorsque que l'on soumet une image via le formulaire, on enregistre cette image de base sans la retoucher. Ensuite, pour récupérer une image (sur la page d'accueil ou ailleurs) : 
- Soit en TWIG en utilisant le filtre "imagine_filter" 
- Soit en PHP en utilisant la méthode **getUrlOfFilteredImage** du service **liip_imagine.service.filter**

Liip s'occupe de générer et mettre en cache la miniature de l'image automatiquement.

Reply:
C'est pour réduire la taille des images uploadées ?

Reply:
Comme vu ensemble c'est pour "pré" générer les miniatures si on a un nouveau format pour empêcher les out of memory lors d'un chargement de page. C'est au cas où vu que j'avais eu le cas en local lors de la première génération;